lib/media/ver20/get_audio_encoder_configuration.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Onvif.Media.Ver20.GetAudioEncoderConfiguration do
+  import SweetXml
+  import XmlBuilder
+
+  alias Onvif.Device
+  alias Onvif.Media.Ver10.Profile.AudioEncoderConfiguration
```

Comment:
Shouldn't we use 

```suggestion
  alias Onvif.Media.Ver20.Profile.AudioEncoderConfiguration
```

?